Sources/_StringProcessing/CharacterClass.swift

```diff
@@ -338,12 +338,14 @@ extension AST.Atom {
     switch kind {
     case let .escaped(b): return b.characterClass
 
-    case .any: return .any
-
     case .property:
       // TODO: Would our model type for character classes include
       // this? Or does grapheme-semantic mode complicate that?
       return nil
+      
+    case .any:
+      // 'any' is handled by Compiler.emitAny(), not `CharacterClass`
```

Comment:
Why?

Reply:
Should be a fatal error then? Unfortunately this is trying to make a CharacterClass, and there's an `any` class, so it's weird to just return `nil` from this.